.github/actions/docusaurus-swizzled-warning/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "docusaurus-swizzled-warning",
+    "version": "0.0.0",
+    "description": "",
+    "main": "index.js",
+    "private": true,
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "jNullj",
+    "license": "CC0",
+    "dependencies": {
+        "@actions/core": "^1.10.0",
+        "@actions/github": "^5.1.1",
```

Comment:
If we are adding a package.json with dependencies here, we should register it withe dependabot, like this:

https://github.com/badges/shields/blob/e9b3b505b324df5cbd72740a7376d09e66e5a4a6/.github/dependabot.yml#L33-L41

Reply:
Solved with https://github.com/badges/shields/pull/9467/commits/c77af690195f3e2eb05ee04c828d1de204e6319b